### PR TITLE
Cave Type A tiles

### PR DESCRIPTION
**Pull request by nkoenig**
Add in Cave Type A tiles.

Signed-off-by: Nate Koenig <nate@openrobotics.org>

### CODE REVIEW

File: docker/download_models.sh [hunk 1]
```diff
@@ -106,7 +107,37 @@ ign fuel download --url "https://fuel.ignitionrobotics.org/openrobotics/models/U
...
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Corner 03 Type A" -v 4
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Corner 04 Type A" -v 4
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Elevation 01 Type A" -v 4
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Elevation 02 Type A" -v 4
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Elevation Corner Type A" -v 4
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Elevation Straight Type A" -v 4
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Split Type A" -v 4
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Straight Shift Type A" -v 4
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Straight Type A" -v 4
+ign fuel download --url "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Cave Transition" -v 4
```
Comment (iche033) [inline docker/download_models.sh:131]: `Cave Transition Type A to and from Type B`

File: subt_ign/src/ConnectionHelper.cc [hunk 2]
```diff
@@ -97,7 +97,34 @@ std::map<std::string, std::vector<ignition::math::Vector3d>>
...
+    {"Cave Split Type A", {{0.0, -50.0, 0.0}, {0.0, 50.0, 0.0}}},
+    {"Cave Straight Shift Type A", {{-25.0, 25.0, 0.0}, {25.0, -25.0, 0.0}}},
+    {"Cave Straight Type A", {{0.0, 25.0, 0.0}, {0.0, -25.0, 0.0}}},
+    {"Cave U Turn 01 Type A", {{-25.0, -25.0, 0.0}, {25.0, -25.0, 0.0}}},
+    {"Cave U Turn Elevation Type A", {{-50.0, -25.0, 0.0}, {0.0, -25.0, 25.0}}},
+    {"Cave Vertical Shaft Cantilevered Type A", {{0.0, -5.46392e-07, 10.0}, {0.0, 50.0, 25.0}}},
+    {"Cave Vertical Shaft Straight Bottom Type A", {{0.0, 0.0, 10.0}, {0.0, 25.0, 0.0}, {0.0, -25.0, 0.0}}},
+    {"Cave Vertical Shaft Straight Top Type A", {{0.0, 25.0, 0.0}, {0.0, -25.0, 0.0}, {0.0, 0.0, 0.0}}},
+    {"Cave Vertical Shaft Type A", {{0.0, 0.0, 10.0}, {0.0, 0.0, 25.0}}},
+    {"Cave Vertical Shaft Dead End Type A", {{0.0, 0.0, 10.0}}}
```
Comment (iche033) [inline subt_ign/src/ConnectionHelper.cc:127]: add transition tile

File: docker/download_models.sh (hunk 1 above)
Comment (nkoenig) [reply docker/download_models.sh:131]: Added in b28cceb3ffb9a66236c936815b5b96e0f2857142.
Can you check with Cole that the connection points are correct?

File: subt_ign/src/ConnectionHelper.cc (hunk 2 above)
Comment (nkoenig) [reply subt_ign/src/ConnectionHelper.cc]: > add type A tiles with lights?

Added in 8a31402ba00787fca6183038423a5c71c88c594b

**Pull request merged**